examples/mysql-atomic-transaction/mysql_atomic_transaction.md

```diff
@@ -1,13 +1,13 @@
 # Database Access - Atomic transactions
 
-This BBE demonstrates how to use the MySQL client to execute a batch of DDL/DML operations with the help of a `transaction` to achieve the atomic behaviour.
+The `mysql` client supports atomic units of work with multiple SQL statements. You can use the Ballerina `transaction` package with the `mysql` client to achieve atomic database transactions. The database makes all changes permanent when the transaction is committed or undo all changes when the transaction is rolled back.
 
 > **Tip**: Checkout [`ballerinax/mssql`](https://central.ballerina.io/ballerinax/mssql), [`ballerinax/postgresql`](https://central.ballerina.io/ballerinax/postgresql), [`ballerinax/oracledb`](https://central.ballerina.io/ballerinax/oracledb), [`ballerinax/java.jdbc`](https://central.ballerina.io/ballerinax/java.jdbc) for other supported database clients.
 
 ::: code mysql_atomic_transaction.bal :::
 
 ## Prerequisite
-- Refer [`mysql-prerequisite`](https://github.com/ballerina-platform/ballerina-distribution/blob/master/examples/mysql-prerequisite/README.md).
+- For more information, see the [Database Access Ballerina By Example - Prerequisites](https://github.com/ballerina-platform/ballerina-distribution/tree/master/examples/mysql-prerequisite).
```

Comment:
Shouldn't be under `Related Links`

Reply:
This will lead to the setup of database files. Wouldn't the user know that if we add it under related links?

Reply:
On second thoughts tis does look like a related link rather than prerequisite. I have tweaked it a bit. Please check.